charts/argo-rollouts/README.md

```diff
@@ -29,20 +29,34 @@ $ helm install --name my-release argo/argo-rollouts
-| controller.metrics.serviceMonitor.enabled | bool | `false` |  |
-| imagePullSecrets | list | `[]` |  |
-| installCRDs | bool | `true` |  |
-| crdAnnotations | object | `{}` |  |
-| podAnnotations | object | `{}` |  |
-| podLabels | object | `{}` |  |
-| serviceAccount.name | string | `"argo-rollouts"` |  |
+| clusterInstall | bool | `true` | `false` runs controller in namespaced mode (does not require cluster RBAC) |
+| controller.component | string | `"rollouts-controller"` | Value of label `app.kubernetes.io/component` |
+| controller.image.pullPolicy | string | `"IfNotPresent"` | Image pull policy |
+| controller.image.registry | string | `quay.io` | Registry to use |
+| controller.image.repository | string | `"argoproj/argo-rollouts"` | Repository to use |
+| controller.image.tag | string | `""` | Overrides the image tag (default is the chart appVersion) |
+| controller.resources | object | `{}` | Resource limits and requests for the controller pods. |
+| controller.tolerations | list | `[]` | [Tolerations for use with node taints](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/) |
+| controller.affinity | object | `{}` | [Assign custom affinity rules to the deployment](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/) |
+| controller.nodeSelector | object | `{}` | [Node selector](https://kubernetes.io/docs/user-guide/node-selection/) |
+| controller.metrics.serviceMonitor.enabled | bool | `false` | Enable a prometheus ServiceMonitor |
+| controller.metrics.serviceMonitor.additionalAnnotations | object | `{}` | Annotations to be added to the ServiceMonitor |
+| controller.metrics.serviceMonitor.additionalLabels | object | `{}` | Labels to be added to the ServiceMonitor |
+| imagePullSecrets | list | `[]` | Registry secret names as an array |
+| podAnnotations | object | `{}` | Annotations to be added to the Rollout pods |
+| podLabels | object | `{}` | Labels to be added to the Rollout pods |
+| serviceAccount.name | string | `""` | The name of the service account to use. If not set and create is true, a name is generated using the fullname template |
+| podSecurityContext | object | `{"runAsNonRoot": true}` | Security Context to set on pod level |
+| containerSecurityContext | object | `{}` | Security Context to set on container level |
+
+## Upgrading
+
+### To 1.0.0
+
+* This is a breaking change which only supports Helm v3.0.0+ now. If you still use Helm v2, please consider upgrading because v2 is EOL since November 2020.  
+  To migrate to helm v3 please have a look at the [Helm 2to3 Plugin](https://github.com/helm/helm-2to3). This tool will convert the existing ConfigMap used for Tiller to a Secret of type `helm.sh/release.v1`.
+* `quay.io` is the default registry now
+* Breaking parameters update
+  * The `installCRDs` and the `crdAnnotations` values have been removed. CRDs are now only installed from the conventional crds/ directory
```

Comment:
Could you point out where the conventional crds is located and the command to install the crds? Thanks.

Reply:
I thought that this chart applies CRDs and an instance of this CRD, therefore the CRD manifest should rest in `chartRoot/crds`. But that's not the case and I reverted this change :-)